process/sync/metablock.go

```diff
@@ -498,9 +498,9 @@ func (boot *MetaBootstrap) requestHeader(nonce uint64) {
 	}
 }
 
-// getHeaderWithNonce method gets the header with given nonce from pool, if it exist there,
+// getRequestedHeaderIfMissing method gets the header with given nonce from pool, if it exist there,
 // and if not it will be requested from network
-func (boot *MetaBootstrap) getHeaderRequestingIfMissing(nonce uint64) (*block.MetaBlock, error) {
+func (boot *MetaBootstrap) getRequestedHeaderIfMissing(nonce uint64) (*block.MetaBlock, error) {
```

Comment:
May revert this change: it sounds better "getHeaderRequestingIfMissing"

Reply:
Done